react/features/chat/components/web/Chat.js

```diff
@@ -88,6 +87,37 @@ class Chat extends AbstractChat<Props> {
         );
     }
 
+    /**
+     * Iterates over all the messages and creates nested arrays which hold
+     * consecutive messages sent be the same participant.
+     *
+     * @private
+     * @returns {Array<Array<Object>>}
+     */
+    _getMessagesGroupedBySender() {
```

Comment:
Why do you have it twice? A leftover from a refactor?

Reply:
Looks like leftover. Initially the web implementation had the grouping done in Chat.js. But to make web more like mobile this logic got moved over to MessageContainer. I'm pretty sure this instance of _getMessagesGroupedBySender doesn't get called.